swing/src/main/java/at/bestsolution/fxembed/swing/FXEmbed.java

```diff
@@ -291,10 +298,13 @@ public void dispose() {
 				List<Stage> list = STAGES.get(window);
 				if (list != null) {
 					list.remove(stage);
+					window.removeComponentListener(componentListener);
 				}
 			}
 			getParent().remove(this);
 			this.fxHandle = 0;
+			
+			stage = null;
```

Comment:
maybe we should also `null` `sceneListener` because we are at it but in general I think the componentListener has been the problem

Reply:
pushed that just now